site/content/quickstart/docker.md

```diff
@@ -266,18 +266,14 @@ curl {{% apiendpoint %}}placement | jq .
 [Read more about the bootstrapping process](/docs/operational_guide/bootstrapping_crash_recovery/).
 {{% /notice %}}
 
-### Readying a Namespace
-
-Once a namespace has finished bootstrapping, it must be marked as ready before receiving traffic. This can be done by calling the _{{% apiendpoint %}}namespace/ready_.
+### Ready a Namespace
+<!-- TODO: Why?> -->
```

Comment:
For context, marking a namespace as ready allows the coordinator to know that it's ready for use. This takes the place of what was required before, namely, updating the coordinator config and restarting every node.